code/game/machinery/sheetifier.dm

```diff
@@ -42,3 +42,14 @@
 	update_icon()
 	var/datum/component/material_container/materials = GetComponent(/datum/component/material_container)
 	materials.retrieve_all() //Returns all as sheets
+
+/obj/machinery/sheetifier/attackby(obj/item/I, mob/user, params)
+	if(default_unfasten_wrench(user, I))
+		return
+	if(default_deconstruction_screwdriver(user, I))
+		update_icon()
+		return
+	if(default_deconstruction_crowbar(I))
+		return
+	else
+		return ..()
```

Comment:
```suggestion
	return ..()

```
Leave a new line at the end, for better git blame preserving.
Since there's a `return` there, the `else` is optional.

Reply:
Committed that, thanks for the tip, I'll be sure to remember it!